Dockerfile-production

```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```

Comment:
I'm just shooting from the hip here, but I *think* that was only required for older version of Node. < 0.10.x?? I haven't looked into that in a while though.